config/crd/bases/cluster.ytsaurus.tech_remoteexecnodes.yaml

```diff
@@ -896,6 +901,14 @@ spec:
                       description: Disk space quota, default is size of related volume.
                       pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                       x-kubernetes-int-or-string: true
+                    watermark:
+                      default: 10
+                      description: Location watermark threshold in percents of volume
```

Comment:
Could you please write a couple of more words (e.g. ", after exceeding which ... happens")? Even after 9 years in YT I struggle to remember what does this watermark affect, and I believe this notion is confusing for almost everybody else. 

Also, I think it is expressed in percents not of volume, bur rather of free volume.

Reply:
What about such description:
> Threshold, as a percentage of volume capacity, for calculation the location's low_watermark, high_watermark, disable_writes_watermark and trash_cleanup_watermark.

The abovementioned watermarks are defined in bytes in YT configs, therefore this threshold is defined as a percentage of volume capacity.
That is, `low_watermark` will be 10% of the volume capacity (by default `watermark = 10`).

I don't think this description is a right place for detailed information about each watermark. Rather it should be in the YTsaurus documentation.
This comments from code looks fine to start with:
```cpp
//! A currently full location is considered to be non-full again when available space grows
//! above this limit.
i64 LowWatermark;

//! A location is considered to be full when available space becomes less than #HighWatermark.
i64 HighWatermark;

//! All writes to the location are aborted when available space becomes less than #DisableWritesWatermark.
i64 DisableWritesWatermark;

//! Maximum amount of time files of a deleted chunk could rest in trash directory before
//! being permanently removed.
TDuration MaxTrashTtl;

//! When free space drops below this watermark, the system starts deleting files in trash directory,
//! starting from the eldest ones.
i64 TrashCleanupWatermark;
```

Reply:
I changed the description to a more understandable one, but to fit into the 80 character limit.